### DISCUSSION TITLE
Does storybook support figma

### DISCUSSION THREAD

**Discussion by Zasvieeel** [storybookjs/storybook#18777] [Help] [answered]
Hello, i want to start disscussion about story that can implementing with figma?

**Comment by rahXephonz** [comment, marked as answer]
I think it supported as their mention on figma website here we go
https://help.figma.com/hc/en-us/articles/360045003494-Storybook-and-Figma

**Reply by Zasvieeel** [reply]
oh nice, thanks for answer my question.

**Reply by rahXephonz** [reply]
yep
